tests/factories/dictionaries.py

```diff
@@ -0,0 +1,33 @@
+import factory
+
```

Comment:
What do you think of the naming of this file? :smiley:

Reply:
I reckon they should live in the same `factories.py` file like the rest

Reply:
hmm, I like them being separated, easier to read each file

Reply:
Not that easy to name them, uh? :D